_applications/equoid.adoc

```diff
@@ -0,0 +1,206 @@
+
+....
+oc new-app --template=oshinko-scala-spark-build-dc \
+    -l app=handler \
+    -p SBT_ARGS=assembly \
+    -p APPLICATION_NAME=equoid-data-handler \
+    -p GIT_URI=https://github.com/radanalyticsio/equoid-data-handler \
+    -p APP_MAIN_CLASS=io.radanalytics.equoid.DataHandler \
+    -e INFINISPAN_HOST=<YOUR_HOSTNAME> \
+    -e INFINISPAN_PORT=<YOUR_PORT> \
+    -e WINDOW_SECONDS=<YOUR_WINDOW_SIZE> \
+    -e SLIDE_SECONDS=<YOUR_SLIDING_SIZE> \
+    -e BATCH_SECONDS=<YOUR_BATCH_SIZE> \
+    -e OP_MODE={single|dual|linear} \
+    -p SPARK_OPTIONS='--driver-java-options=-Dvertx.cacheDirBase=/tmp'
+....
+
+=== Launch Web UI and Set Up Keycloak
+
+Equoid's web UI is launched using a script developed by one of Equoid's contributors, Jiri Kremser. The following calls will utilize that script to set up the UI as well as a Keycloak instance for users and login. Finally, an edit role is given to the project's service account so additional data handler instances can be created/modified. 
+
+....
+BASE_URL="https://raw.githubusercontent.com/radanalyticsio/equoid-ui/master/ocp/"
+curl -sSL $BASE_URL/ocp-apply.sh | \
+    BASE_URL="$BASE_URL" \
+    KC_REALM_PATH="web-ui/keycloak/realm-config" \
+    bash -s stable
+oc policy add-role-to-user edit system:serviceaccount:`oc project -q`:default
+....
+
+[[usage]]
+== Usage
+
+=== Web Interface
+With all the microservices running, you can now view a visualization of the data stream's top-ks for the parameters you set. By browsing to the OpenShift webui for your cluster, logging in, and looking at your project's overview, you should see a pod/application named equoid-ui with an associated URI. By clicking on the latter, you will then be at the login screen for your equoid system. Log in using either of the default credential sets listed on the login page, then click on the graph tab. You should see a visualization of your stream's data which updates periodically. 
+
+In addition, you can add a frequent item by clicking on the add item button. Finally, you can add another data handler via the command line, a script is included in equoid-openshift to facilitate additional data handler instances. Once these are up and running, the web UI will automatically add their data in a separate graph tab. 
+ 
+=== Equoid Application Settings
+We provide a number of variables to modify per the needs of end users. In this section, these are described. 
```

Comment:
Could simplify for ease of reading?
Equoid's tuning variables are described below:

Reply:
Good point, will address in revision